src/components/views/elements/EventTilePreview.tsx

```diff
@@ -118,13 +119,15 @@ export default class EventTilePreview extends React.Component<IProps, IState> {
     }
 
     public render() {
-        const event = this.fakeEvent(this.state);
-
         const className = classnames(this.props.className, {
             "mx_IRCLayout": this.props.layout == Layout.IRC,
             "mx_GroupLayout": this.props.layout == Layout.Group,
         });
```

Comment:
```suggestion
        const className = classnames(this.props.className, {
            "mx_IRCLayout": this.props.layout == Layout.IRC,
            "mx_GroupLayout": this.props.layout == Layout.Group,
            "mx_EventTilePreview_loader": !this.props.userId,
        });
```

I think this would be a bit nicer and you don't have to have the inline `classnames` then